package.json

```diff
@@ -18,6 +18,11 @@
       "require": "./dist/index.js",
       "types": "./dist/types/index.d.ts"
     },
+    "./node": {
```

Comment:
Good idea! We should do this for more packages that have some Node.js-only functions.